frontend/src/pages/Landing/LandingPage.module.scss

```diff
@@ -11,14 +11,15 @@
 .PrimaryButton {
   background-color: #0b5240;
   color: white !important;
-  border-radius: 40px;
-  height: 52px;
```

Comment:
Verifying these changes rq too -- these are preserved in the Material Button stylings?

Reply:
The only styles that I removed from CSS were the height and width since material will calculate those automatically to be the right size. So if someone changed the text later they wouldn't have to update the height and width.